components/suggestion/search_suggestion_tagged_input.jsx

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SearchSuggestionInput from './search_suggestion_input';
+const exampleTags = ['from:', 'in:', 'on:', 'before:', 'after:'];
+
+export default class SearchSuggestionTaggedInput extends SearchSuggestionInput {
```

Comment:
Also, I'm really hesitant about having this extend `SearchSuggestionInput`. Inheritance isn't recommended with React, and whenever I've tried using it in the past, it's caused problems.